go.mod

```diff
@@ -63,6 +63,11 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
+require (
```

Comment:
Nit: but can move this back into the block? They're the same version. Why doesn't go have a standard way to format mod files seems strange

Reply:
I avoid touching the go.mod file and just live with what ever way Go chooses to format things for me :p